src/SAWScript/Crucible/MIR/Setup/Value.hs

```diff
@@ -157,3 +229,137 @@ data MirSetupSlice
 makeLenses ''MIRCrucibleContext
 makeLenses ''MirAllocSpec
 makeLenses ''MirPointer
+
+{-
+Note [Symbolic enums]
```

Comment:
A large piece of documentation that I wrote about the approach to handling symbolic enum values.